clients/web/src/templates/body/itemPage.pug

```diff
@@ -48,6 +48,10 @@
   .g-item-updated.g-info-list-entry
     i.icon-clock
     | Updated on #{formatDate(item.get('updated'), DATE_SECOND)}
+  .g-item-info-line.g-info-list-entry(property="id")
+    i.icon-share
+    span Unique ID:
+      span.g-bold-part #{item.id}
```

Comment:
This looks like this when rendered:

![screen shot 2017-07-27 at 5 26 49 pm](https://user-images.githubusercontent.com/555959/28692975-da13f494-72f0-11e7-8dd6-8287c5c11b60.png)

We should make sure there's a space between the elements, and also remove the unnecessary stuff that was copy/pasted from the folder info dialog.

```
.g-item-id.g-info-list-entry
   i.icon-share
   | Unique ID: #[b= item.id]
```

Or if we think it looks better without the bold text, just


```
.g-item-id.g-info-list-entry
   i.icon-share
   | Unique ID: #{item.id}
```